web_translate_dialog/static/src/js/web_translate_dialog.js

```diff
@@ -0,0 +1,224 @@
+            $(this).toggleClass('touched', $(this).val() !== $(this).attr('data-value'));
+        });
+        this.$footer.html(QWeb.render("TranslateDialog.buttons"));
+        this.$footer.find(".oe_form_translate_dialog_save_button").click(function(){
+            self.on_button_save();
+            self.on_button_close();
+        });
+        this.$footer.find(".oe_form_translate_dialog_cancel_button").click(function(){
+            self.on_button_close();
+        });
+
+        this.do_load_fields_values();
+    },
+    initialize_html_fields: function(lang) {
+        var self = this;
+        _.each(this.translatable_fields, function(f) {
+            // Initialize summernote if HTML field
+            self.$el.find('.oe_form_field_html .oe_translation_field[name="' + lang.code + '-' + f + '"]').each(function() {
+                var $parent = $(this).summernote({
+                    'focus': false,
+                    'toolbar': [
+                        ['style', ['style']],
+                        ['font', ['bold', 'italic', 'underline', 'clear']],
+                        ['fontsize', ['fontsize']],
+                        ['color', ['color']],
+                        ['para', ['ul', 'ol', 'paragraph']],
+                        ['table', ['table']],
+                        ['insert', ['link', 'picture']],
+                        ['misc', ['codeview']],
+                        ['history', ['undo', 'redo']]
+                    ],
+                    'prettifyHtml': false,
+                    'styleWithSpan': false,
+                    'inlinemedia': ['p'],
+                    'lang': "odoo",
+                    'onChange': function (value) {
+                        $(this).toggleClass('touched', value !== $(this).attr('data-value'));
+                    }
+                }).parent();
+                // Triggers a mouseup to refresh the editor toolbar
+                $parent.find('.note-editable').trigger('mouseup');
+                $parent.find('.note-editing-area').css({
+                    minHeight:'100px',
+                    minWidth:'260px',
+                });
+            });
+        });
+    },
+    set_fields_values: function(lang, values) {
+        var self = this;
+        _.each(this.translatable_fields, function(f) {
+            self.$el.find('.oe_translation_field[name="' + lang.code +
+                    '-' + f + '"]').val(values[f] || '').attr(
+                    'data-value', values[f] || '');
+        });
+        this.$el.find('textarea.oe_translation_field').css({
+            minHeight:'100px',
+        });
+        $(window).resize();
+        this.initialize_html_fields(lang);
+    },
+    do_load_fields_values: function() {
+        var self = this,
+            deferred = [];
+
+        this.$el.find('.oe_translation_field').val('').removeClass('touched');
+        _.each(self.languages, function(lg) {
+            var context = session.user_context;
```

Comment:
As discussed together, we want to change this line to create a new context for each language.